### ISSUE TITLE
Better error message for #522

### ISSUE THREAD

**Issue by gaborcsardi** [r-lib/cli#523]

**Comment by gaborcsardi** [comment]
#526 fixes this:
```r
❯ cli::cli_text("xxx {.foobar} yyy")
Error in `glue(str, .envir = .envir, .transformer = transformer, .cli = TRUE)` at inline.R:324:3:
! Invalid cli literal: `{.foobar}` starts with a dot.
ℹ Interpreted literals must not start with a dot in cli >= 3.4.0.
ℹ `{}` expressions starting with a dot are now only used for cli styles.
ℹ To avoid this error, put a space character after the starting `{` or use parentheses:
  `{(.foobar)}`.
Type .Last.error to see the more details.
```

**Comment by gaborcsardi** [comment]
Closed by #526.
